lib/events/discard.go

```diff
@@ -91,7 +91,7 @@ func (*DiscardStream) Complete(ctx context.Context) error {
 
 // EmitAuditEvent discards audit event
 func (*DiscardStream) EmitAuditEvent(ctx context.Context, event apievents.AuditEvent) error {
-	log.Debugf("Dicarding stream event: %v", event)
+	log.Debugf("Discarding stream event: id=%s, type=%s, time=%s, index=%s", event.GetID(), event.GetType(), event.GetTime(), event.GetIndex())
```

Comment:
We could probably use `WithFields` here, so this is nicer when the logger is running with JSON output.

Reply:
Thanks for the suggestion @strideynet. Done in 2464adee2eb4376ee1811a463b4133ce5ea596ca.